css-grid-1/grid-items/grid-item-containing-block-001.xht

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <title>CSS Grid Layout Test: A grid item is sized within the containing block defined by its grid area</title>
+        <link rel="author" title="Tomek Wytrebowicz" href="mailto:tomalecpub@gmail.com" />
+        <link rel="help" href="http://www.w3.org/TR/css-grid-1/#grid-items" title="4. A grid item is sized within the containing block defined by its grid area" />
+        <link rel="match" href="../../css21/reference/ref-filled-green-100px-square.xht" />
+        <style type="text/css"><![CDATA[
+            #grid {
+                display: grid;
+                width: 200px;
+                height: 200px;
+                grid-template-rows: 100px 100px;
+                grid-template-columns: 100px 100px;
```

Comment:
Yeah I understand the width and height 200px. I was thinking that for the grid itself we can use:
  grid-template-rows: 100px;
  grid-template-columns: 100px;

As we're just placing the item in the first row and first column, so we don't need a 2x2 grid.